Ibidapo/iReporter-UI
# feat(registration): build join-us page

#### What does this PR do?
This PR builds the Join Us page, where users can register.

#### Description of Task to be completed?
N/A

#### How should this be manually tested?
- Ensure that API is running
- Checkout to this branch
- run `npm start`
- log onto `localhost:3000`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#169565739](https://www.pivotaltracker.com/story/show/169565739)
